README.md

```diff
@@ -1,18 +1,36 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Payment](#payment)
+  - [General Info](#general-info)
+  - [Compiling](#compiling)
+  - [Integration into the shop:](#integration-into-the-shop)
+    - [Maven example](#maven-example)
+    - [SBT example](#sbt-example)
+    - [Gradle example](#gradle-example)
+    - [Getting the payment methods](#getting-the-payment-methods)
+    - [Creating the Payment](#creating-the-payment)
+    - [Creating the PaymentTransaction](#creating-the-paymenttransaction)
+  - [Supported payment methods](#supported-payment-methods)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
 Payment
 ==================
 
-[![Build Status](https://travis-ci.com/commercetools/project-payment.svg?token=xBfuKwz4mtssFw4UENNy&branch=master)](https://travis-ci.com/commercetools/project-payment) 
-[![Stories in Ready](https://badge.waffle.io/commercetools/project-payment.png?label=ready&title=Ready)](https://waffle.io/commercetools/project-payment)
+[![Build Status](https://travis-ci.com/commercetools/commercetools-payment-integration-java.svg?token=xBfuKwz4mtssFw4UENNy&branch=master)](https://travis-ci.com/commercetools/project-payment) 
+[![Stories in Ready](https://badge.waffle.io/commercetools/commercetools-payment-integration-java.png?label=ready&title=Ready)](https://waffle.io/commercetools/project-payment)
 
 Java modules to create and handle payments and payment transactions in different _payment integration services (providers)_, 
 like [`commercetools Payone Integration Service`](https://github.com/commercetools/commercetools-payone-integration). 
 
-* [Javadoc](https://commercetools.github.io/project-payment/javadoc/index.html)
+* [Javadoc](https://commercetools.github.io/commercetools-payment-integration-java/javadoc/v/)
 
 * [Build and publish workflow](BUILD.md)
 
 ## General Info
-The project-payment project intend is to make payment integration easy.
+The `commercetools-payment-integration-java` project intend is to make payment integration easy.
```

Comment:
`This projects provides collection of utils which supports payment integrations based on intermediary services like [Payone Integration Service](https://github.com/commercetools/commercetools-payone-integration)`